www/blog/2023/06/19/karmashapes/index.md

```diff
@@ -0,0 +1,790 @@
+  </tr>
+  <tr>
+   <td><em>Sub-district</em>
+   </td>
+   <td><em>N/A</em>
+   </td>
+   <td><em>Speculative new WOF placetype, a followup to the Karmashapes import</em>
+   </td>
+  </tr>
+  <tr>
+   <td><em>Village</em>
+   </td>
+   <td><em>Localadmin and locality</em>
+   </td>
+   <td><em>Significant WOF upgrade post-Karmashapes</em>
+   </td>
+  </tr>
+  <tr>
+   <td><em>Urban</em>
+   </td>
+   <td><em>Localadmin and locality (sometimes coterminous with county)</em>
+   </td>
+   <td><em>In WOF pre-Karmashapes</em>
+   </td>
+  </tr>
+</table>
+
+
+Additional localization work was done in WOF once Karmashapes geometries and properties were imported. Hindi translations were added to any locality record with a population of 50,000 or more and additional localizations were added based on regional language patterns. WOF maintains official and spoken language by region, so each locality’s parent region was evaluated to prompt additional relevant localizations to match local expectations and supplement the Hindi localizations. In total, name translations were added to over 17,000 records over what was provided by Karmashapes.
+
+Using openly-licensed data at each step of the build process allows Karmashapes to be released under a simple CC-BY license and, in turn, allows Who’s On First to import the full Karmashapes dataset. Without openly licensed data, this build would not have been possible.
+
+# Get the Data
+
+Karmashapes and Who’s On First data are available in several file formats and through various endpoints. You can access and browse data here:
+
+* [Browse](https://spelunker.whosonfirst.org/placetypes/locality/?&iso=in) Who’s On First records in the Spelunker
+* [Download](https://geocode.earth/data/whosonfirst/#IN) Who’s On First data in Shapefile and SQLite formats, via Geocode.Earth: 
+* [Learn](https://whosonfirst.org/blog/2023/05/31/shapefiles/) more about downloading Who’s On First data in various formats
+* Want the raw Karmashapes files? Full Karmashapes downloads are [available](https://karmashapes.wordpress.com/) on Justin’s blog. 
+
+# Next Steps
+
+Because the village and habitation data in India often overlaps, especially near the largest metro areas in the country, Karmashapes records were imported into Who’s On First most often as locality records and sometimes as neighbourhood records when they fell in an existing WOF locality. Practically speaking, this means many of the Who’s On First records in and around the ~1,500 largest metropolitan areas in India should be reviewed.
+
+Some of the input sources used to create Karmashapes use population estimates from before the last census and those vintage population values for localities and neighbourhoods may also need to be updated.
+
+[![](images/images/next-steps-1.png)]()
+![Next Steps](images/next-steps-1.png "Next Steps")
+_(above) Ahmedabad locality extent with curated neighbourhood boundaries and a few new, funky Karamashapes neighbourhoods on the margins_
```

Comment:
To: `and a few new, funky Karamashapes neighbourhoods on the margins in darker blue`

From: `and a few new, funky Karamashapes neighbourhoods on the margins`